pydantic_core/core_schema.py

```diff
@@ -415,23 +415,25 @@ def model_ser_schema(cls: Type[Any], schema: CoreSchema) -> ModelSerSchema:
 class ComputedField(TypedDict, total=False):
     type: Required[Literal['computed-field']]
     property_name: Required[str]
-    json_return_type: JsonReturnTypes
+    return_schema: Required[CoreSchema]
```

Comment:
I assume we need to change Pydantic to generate this from a return type annotation (which we'll require)?

Reply:
yes, see https://github.com/pydantic/pydantic/pull/5708